src/pretix/settings.py

```diff
@@ -373,6 +373,8 @@
 }
 
 MIDDLEWARE = [
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
You should remove this middleware to test if the Nginx config works.

Reply:
Done 👍